upgrade/capsule.py

```diff
@@ -7,12 +7,25 @@
 from fabric.api import execute
 from fabric.api import run
 
+from upgrade.helpers.constants.environment_config import CAPSULE_HOSTNAME
```

Comment:
This is applicable for all env vars here.

Reply:
I do not want to access the variable everywhere by using settings objects, that's why, I created a separate file and mentioned what variable we are using in the upgrade code base, It will help me to make the code clean and manageable.